src/components/WwwFrame/TheHeader.vue

```diff
@@ -1,275 +1,290 @@
 <template>
-	<header class="top-nav">
-		<div class="header-row row">
-			<router-link class="header-logo header-button" to="/" v-kv-track-event="['TopNav','click-Logo']">
-				<kv-icon name="new-kiva-logo" />
-			</router-link>
-			<router-link
-				:id="lendMenuId"
-				to="/lend-by-category"
-				class="header-button"
-				v-kv-track-event="['TopNav','click-Lend']"
-			>
-				<span>Lend <kv-icon name="triangle" /></span>
-			</router-link>
-			<button class="search-toggler header-button"
-				:class="{'show-for-large': isVisitor}"
-				:aria-expanded="searchOpen ? 'true' : 'false'"
-				:aria-pressed="searchOpen ? 'true' : 'false'"
-				aria-controls="top-nav-search-area"
-				v-show="!hideSearchInHeader"
-				@click="toggleSearch"
-				v-kv-track-event="['TopNav','click-search-toggle']"
-			>
-				<kv-icon v-show="!searchOpen" class="search-icon" name="magnify-glass" />
-				<kv-icon v-show="searchOpen" class="close-icon" name="x" />
-			</button>
-			<div class="flexible-center-area">
-				<div id="top-nav-search-area" v-if="!hideSearchInHeader" :aria-hidden="searchOpen ? 'false' : 'true'">
-					<button class="close-search hide-for-large"
-						:aria-expanded="searchOpen ? 'true' : 'false'"
-						:aria-pressed="searchOpen ? 'true' : 'false'"
-						aria-controls="top-nav-search-area"
-						@click="toggleSearch"
-						v-kv-track-event="['TopNav','click-search-close-mobile']"
+	<header class="top-nav" :style="cssVars">
+		<nav aria-label="Primary navigation">
```

Comment:
This diff of the template seems like a lot, but the change is wrapping everything in the header with a <nav> element